scico/linop/xray/astra.py

```diff
@@ -604,12 +620,15 @@ def __init__(
 
         if not isinstance(det_count, (list, tuple)) or len(det_count) != 2:
             raise ValueError("Expected det_count to be a tuple with 2 elements.")
+        if angles is not None and vectors is not None:
+            raise ValueError("`angles` and `vectors` are mutually exclusive.")
```

Comment:
Perhaps "Parameters angles and vectors ..."? Also, exceptions typically go to a text console, so no point in adding markup.

Reply:
Understood about error messages going to the console, but I do think it's useful to have some marker that we are talking about a parameter name. E.g., numpy uses single quotes.

```
 sum() missing 1 required positional argument: 'a'
```

Reply:
Fair point. But for consistency we should make a note (perhaps a new issue) to also apply that principle elsewhere in the code.

Reply:
Issue #583 created.